tests/test_edge_cases.py

```diff
@@ -1166,6 +1167,7 @@ def test_unable_to_infer():
         class InvalidDefinitionModel(BaseModel):
             x = None
 
+    # TODO: Do we want any changes to this behavior in v2? (Currently, no error is raised)
     assert exc_info.value.args[0] == 'unable to infer type for attribute "x"'
```

Comment:
Question 2 ☝️

Reply:
`x` definitely shouldn't be a field, I guess an error would be best, but might be hard to identify "non-field attributes reliable"?

Reply:
Would the error be that `x` is a non-field attribute? Wouldn't the error just be something along the lines of `x does not have an annotation`?

Reply:
yes, it should be changed to something like that; however I don't want to update the expected error message here, I think that can be done in whatever pull request makes it actually raise an error.